winui/Masked-TextBox/Events.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Events in WinUI MaskedTextBox control | Syncfusion
+description: Learn all about the events in the MaskedTextBox (SfMaskedTextBox) control.
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# Events in WinUI MaskedTextBox
+
+This section explains the events available in the WinUI MaskedTextBox control.
+
+## ValueChanging
+
+The ValueChanging event occurs when the [Value](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfMaskedTextBox.html#Syncfusion_UI_Xaml_Editors_SfMaskedTextBox_Value) property is changing in the MaskedTextBox control. 
+
+The MaskedTextBoxValueChangingEventArgs object contains the following properties:
+
+* NewValue: Returns the current value of the MaskedTextBox control.
+* OldValue: Returns the previous value of the MaskedTextBox control.
+* IsValid: Gets or sets a boolean value indicating whether the input is considered valid according to mask completion.
+* Cancel: Gets or sets a value indicating whether the event should be canceled.
+
+## ValueChanged
+
+The [ValueChanged](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfMaskedTextBox.html#Syncfusion_UI_Xaml_Editors_SfMaskedTextBox_ValueChanged) event occurs when the [Value](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfMaskedTextBox.html#Syncfusion_UI_Xaml_Editors_SfMaskedTextBox_Value) property is changed in the MaskedTextBox control. 
```

Comment:
@BhaskarSuresh22 

## ValueChanged Event

The ValueChanged event occurs when the Value property is changed in the MaskedTextBox control. The MaskedTextBoxValueChangedEventArgs provides the following properties:

IsMaskCompleted: Gets a boolean value indicating whether all the required inputs for the mask are completed.
NewValue: Gets the current value of the MaskedTextBox control.
OldValue: Gets the previous value of the MaskedTextBox control.

private void MaskedTextBox_ValueChanged(object sender, MaskedTextBoxValueChangedEventArgs e)
{
    // Access the new and old values
    string newValue = e.NewValue;
    string oldValue = e.OldValue;

    // Check if the mask input is completed
    bool isMaskCompleted = e.IsMaskCompleted;

    // Perform actions based on the value change
    // ...
}

Reply:
@Manivannan-E, Done the changes